tests/subsys/storage/flash_map/src/main.c

```diff
@@ -32,51 +32,96 @@ ZTEST(flash_map, test_flash_area_disabled_device)
 	zassert_equal(rc, -ENODEV, "Open did not fail");
 }
 
+ZTEST(flash_map, test_flash_area_device_is_ready)
+{
+	const struct flash_area no_dev {
+		.fa_dev = NULL;
+	};
+
+	zassert_false(test_flash_area_device_is_ready(NULL));
+	zassert_false(test_flash_area_device_is_ready(no_dev));
+	/* The below just assumes that tests are executed so late that
+	 * all devices are already initialized and ready.
+	 */
+	zassert_true(test_flash_area_device_is_ready(
+			FIXED_PARTITION(SLOT1_PARTITION));
```

Comment:
```suggestion
	zassert_false(flash_area_device_is_ready(NULL));
	zassert_false(flash_area_device_is_ready(no_dev));
	/* The below just assumes that tests are executed so late that
	 * all devices are already initialized and ready.
	 */
	zassert_true(flash_area_device_is_ready(
			FIXED_PARTITION(SLOT1_PARTITION));
```

Reply:
Thanks. I am so glad this is my mistake, because I have spent yesterday an hour on battle between clang-format and compliance check and gave up.